src/app/components/product-table/product-table.component.scss

```diff
@@ -25,14 +22,79 @@ td.mat-cell {
 }
 
 .loading-shade {
-  position: absolute;
-  top: 0;
-  left: 0;
+  align-items: center;
+  background: rgba(0, 0, 0, 0.15);
   bottom: 0;
+  display: flex;
+  justify-content: center;
+  left: 0;
+  position: absolute;
   right: 0;
-  background: rgba(0, 0, 0, 0.15);
+  top: 0;
   z-index: 1;
-  display: flex;
+}
+
+.description {
+  box-sizing: border-box;
+  white-space: pre-line;
+  word-wrap: break-word;
+
+  p {
+    -webkit-box-orient: vertical;
+    display: -webkit-box;
```

Comment:
what is this display for ? I couldn't find it in documentation.

Reply:
Hey, I checked it out once again, as I'd previously said it's the old way of doing flex before it was supported, and `-webkit-line-clamp` only works coupled with these prefixes, `line-clamp` is in draft still and not supported
https://drafts.csswg.org/css-overflow-3/#webkit-line-clamp

So I switched to set height and `before` pseud-element